vignettes/download_github_comments.Rmd

```diff
@@ -125,7 +131,6 @@ all_issue_files <- lapply(list.files(save_path_issue_refresh, full.names = TRUE)
 # Parse each JSON file using the refresh parser
 all_issue <- lapply(all_issue_files, github_parse_search_issues_refresh)
 
-
 # Combine all the data tables
 all_issues_combined <- rbindlist(all_issue, fill = TRUE)
 
```

Comment:
For "Issues by Customized Queries", can you explain what parameters they can change? Why or when they would use such a function?